tests/BugsnagUnity.Tests/ConfigurationTests.cs

```diff
@@ -82,5 +83,61 @@ public void EndpointValidation()
 
             Assert.IsFalse(config.Endpoints.IsValid);
         }
+
+        [Test]
+        public void RedactedKeysTest()
+        {
+            var config = new Configuration("foo");
+
+            // Default redacted keys
+            Assert.IsTrue(config.KeyIsRedacted("password"));
```

Comment:
I'd add the extra assertion for non-exact match, as we decided this is what we want:

```
            Assert.IsTrue(config.KeyIsRedacted("mypasswordfield"));
```